components/org.wso2.carbon.identity.oauth.endpoint/src/main/webapp/WEB-INF/cxf-servlet.xml

```diff
@@ -40,6 +40,7 @@
         </jaxrs:serviceBeans>
         <jaxrs:inInterceptors>
             <ref bean="oauthClientAuthnBean"/>
+            <ref bean="clientAttestationBean"/>
```

Comment:
Lets remove this and add to the relevant class as an annotation `@InInterceptors(classes = ClientAttestationProxy.class)`

Reply:
Added a service holder and remove clientAttestationBean bean